pkg/deployments/tasks/20221205-veboost-v2/test/test.fork.ts

```diff
@@ -0,0 +1,95 @@
+import hre from 'hardhat';
+import { Contract } from 'ethers';
+import { expect } from 'chai';
+
+import { ZERO_BYTES32 } from '@balancer-labs/v2-helpers/src/constants';
+
+import { actionId } from '@balancer-labs/v2-helpers/src/models/misc/actions';
+
+import { describeForkTest, impersonate, getForkedNetwork, Task, TaskMode } from '../../../src';
+
+describeForkTest('veBoostV2', 'mainnet', 16110000, function () {
+  let oldDelegation: Contract;
+  let delegation: Contract;
+
+  let task: Task;
+
+  const GOV_MULTISIG = '0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f';
+
+  const existingBoosts: string[] = [
+    '0x7f01d9b227593e033bf8d6fc86e634d27aa85568000000000000000000000000',
+    '0xc2593e6a71130e7525ec3e64ba7795827086df0a000000000000000000000000',
+    '0xef9a40f0ce782108233b6a5d8fef08c89b01a7bd000000000000000000000000',
+    '0x0035fc5208ef989c28d47e552e92b0c507d2b318000000000000000000000000',
+  ];
```

Comment:
Can we check if this is comprehensive?

Reply:
I've added a check to ensure that this array is equal in length to the total supply of boosts.